plugins/window_size/gtk/window_size_plugin.cc

```diff
@@ -0,0 +1,203 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+#include "include/window_size/window_size_plugin.h"
+
+#include <flutter_linux/flutter_linux.h>
+#include <gtk/gtk.h>
+
+// TODO(robert-ancell): Use GdkMonitor when we can depend on GTK 3.22
```

Comment:
Should we just go ahead and do this now? I'm not sure we need to support below 18.04 in general, which looks like it has 3.22, and certainly for an experimental plugin I'm fine with 18.04+.

Reply:
I think it's a reasonable minimum dependency - I was just copying the existing code.

Reply:
IIRC we were looking at 16.04 as a reasonable baseline at the time the initial Linux work happened, but time marches on :)